docs/starter-kit-ui/starter_kit_ui.md

```diff
@@ -0,0 +1,43 @@
+---
+sidebar_label: Overview
+title: Starter Kit UI Overview
+id: overview
+sidebar_position: 1
+---
+
+## Starter Kit UI: Getting Started
+
+### Configuration
```

Comment:
Please lift the content that's now on the repo README [here](https://github.com/ga4gh/ga4gh-starter-kit-ui) and adapt it for here. You've outlined how to run the docker command with all defaults, but not with custom config file and ports.

We also need a table of what are the configurable properties and their defaults, as is available on the README